client/src/App/index.js

```diff
@@ -3,43 +3,61 @@ import './style.css';
 import {
   Switch,
   Route,
+  useLocation,
 } from 'react-router-dom';
 import { ProvideAuth } from './useAuth';
 import { ProtectedRoute, AdminProtectedRoute } from './ProtectedRoute';
-import Navbar from '../Components/Common/Navbar';
-import routes from './routes';
+import TypeContext from '../TypeContext/TypeContext';
+import Home from '../Pages/Home';
 
 function App() {
+  const location = useLocation();
   return (
     <div className="App">
       <ProvideAuth>
-        <Navbar />
 
         <Switch>
-          {routes.map((route) => {
-            if (route.public) {
-              return <Route exact path={route.path} key={route.name}>{route.children}</Route>;
-            }
-            if (route.isAdmin) {
-              return (
-                <AdminProtectedRoute exact path={route.path} key={route.name}>
-                  {route.children}
-                </AdminProtectedRoute>
-              );
-            }
-            return (
-              <ProtectedRoute exact path={route.path} key={route.name}>
-                {route.children}
-              </ProtectedRoute>
-            );
-          })}
+          <Route exact path="/cars">
```

Comment:
Why not do something like you did with `dashboardRoutes`? With the array thingie?